typing/ctype.ml

```diff
@@ -730,8 +727,10 @@ let duplicate_class_type ty =
 *)
 let rec generalize ty =
   let ty = repr ty in
+  (* generalize the type iff ty.level <= !current_level *)
```

Comment:
I'm a bit late to the party, but I think that comment is incorrect (and also, vaguely useless?).